website/docs/r/maintenance_window.html.markdown

```diff
@@ -47,6 +47,7 @@ Once maintenance is scheduled for your cluster, you cannot change your maintenan
 * `number_of_deferrals` - Number of times the current maintenance event for this project has been deferred, you can set a maximum of 2 deferrals.
 * `defer` - Defer maintenance for the given project for one week.
 * `auto_defer` - Automatically defer any maintenance for the given project for one week.
+* `auto_defer_once_enabled` - Flag that indicates whether you want to defer all maintenance windows one week they would be triggered.
```

Comment:
Just a slight edit here is needed for clarity:
Automatically defers any maintenance for the given project for one week when a maintenance event is available.

Reply:
Okay, I had to recheck this against the original docs - now I see my confusion - one more slight change, including it all here so it makes sense:
* `defer` - Defer the next scheduled maintenance for the given project for one week.
* `auto_defer` - Defer any scheduled maintenance for the given project for one week.
* `auto_defer_once_enabled` - Flag that indicates whether you want to defer all maintenance windows one week they would be triggered.

Reply:
Already changed :)